.dockerignore

```diff
@@ -1,6 +1,9 @@
-*
-!Dockerfile
-!dist
-!install.bash
-!requirements.txt
-!requirements/
+# Directories to ignore from the Docker build context
+# Calling COPY or RUN --mount=type=bind is slow if the build context is large.
+.git/
+app/.yarn/
+build/
+dist/
+fiftyone.egg-info/
+**/node_modules/
```

Comment:
Unlike `.gitignore`, a pattern like `node_modules/` only matches the dir in the root folder. The `**` matches subdirectories as well.